plugins/kotlin/idea/tests/testData/parameterInfo/functionCall/UnmappedCurrent.kt

```diff
@@ -0,0 +1,8 @@
+fun m(x: Int, y: Boolean) = 2
+
+fun d() {
+    m(y = false, <caret>unmapped = false)
+}
+/*
+Text: (<disabled>[y: Boolean],</disabled><highlight> </highlight>[x: Int]), Disabled: false, Strikeout: false, Green: true
```

Comment:
Just curious how to interpret `<highlight> </highlight>` (v.s., e.g., no highliht)?

Reply:
I copied FE 1.0 behavior in certain cases, see comment in: https://github.com/JetBrains/intellij-community/blob/d94d5dc655ae54f2552c84f6ddebb906daf9a864/plugins/kotlin/fir/src/org/jetbrains/kotlin/idea/parameterInfo/KotlinHighLevelFunctionParameterInfoHandler.kt#L334-L343

I find it a little confusing, but my understanding is in order for `isDisabledBeforeHighlight` to work, you have to highlight _something_ even if it's just the empty space. My interpretation of this presentation is that `[y: Boolean]` is "disabled" because it's already used.

I'll update the comment above. When I first wrote it, I didn't understand why the presentation was that way.